gh-pages/assets/javascript/index.js

```diff
@@ -0,0 +1,80 @@
+import utils from "./utils.js"
+import { FrenchMap } from "./map.js"
+
+/**
+ * Populate the table containing prices averages
+ * @param {Map<string, number>} averages Map from fuel_type to price
+ * @param {Map<string, string>} fuelNames Map from fuel_type to fuel_name
```

Comment:
These are not actual `Map<>` objects but `object`s, no ?

Reply:
Done https://github.com/Doreapp/prix-carburants/pull/36/commits/28268a09deb1f94c6a3960ed1f2f7cac70dc05a1